### PR TITLE
Add Globalnet limitations 2.7 stolostron/backlog#26901

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/26901

Also for 2.6.

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -1043,3 +1043,9 @@ Only non-NAT deployments support Submariner deployments with the VXLAN cable dri
 //2.6:25275
 
 Using the OVN Kubernetes CNI network provider requires Red Hat OpenShift 4.11 or later. OVN Kubernetes does not support Globalnet.
+
+[#globalnet-limitations]
+=== Globalnet limitations
+//2.6:26901
+
+Globalnet is not supported with Red Hat OpenShift Data Foundation disaster recovery solutions. Make sure to use a separate range of private IP addresses in each cluster for multi-cluster disaster recovery scenarios.
```
Comment (swopebe) [inline release_notes/known_issues.adoc]: multicluster...

(If this stays as a known issue)

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -1043,3 +1043,9 @@ Only non-NAT deployments support Submariner deployments with the VXLAN cable dri
 //2.6:25275
 
 Using the OVN Kubernetes CNI network provider requires Red Hat OpenShift 4.11 or later. OVN Kubernetes does not support Globalnet.
+
+[#globalnet-limitations]
+=== Globalnet limitations
+//2.6:26901
+
+Globalnet is not supported with Red Hat OpenShift Data Foundation disaster recovery solutions. Make sure to use a separate range of private IP addresses in each cluster for multicluster disaster recovery scenarios.
```
Comment (olivelakra) [inline release_notes/known_issues.adoc]: Globalnet is not supported with Red Hat OpenShift Data Foundation disaster recovery solutions currently. Make sure to use a non-overlapping range of private IP addresses for the cluster and service networks in each cluster for regional disaster recovery scenarios.

Comment (olivelakra) [reply release_notes/known_issues.adoc]: this note applies to ACM 2.5, ACM 2.6 and ACM 2.7 docs

**Comment by olivelakra** [comment]
@oafischer updates are LGTM

**Pull request merged**